### DISCUSSION TITLE
Gruntwork module for AWS Network Firewall

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#481] [Help] [answered]
AWS Network Firewall is a relatively new service: https://aws.amazon.com/about-aws/whats-new/2020/11/introducing-aws-network-firewall/.

Does Gruntwork have a module for this that I can use? 

---

<ins datetime="2022-06-22T16:58:37Z">
  <p><a href="https://support.gruntwork.io/hc/requests/108816">Tracked in ticket #108816</a></p>
</ins>

**Comment by yorinasub17** [comment, marked as answer]
This is on our roadmap to implement, but hasn't been prioritized. You can follow along on [this PR](https://github.com/gruntwork-io/terraform-aws-vpc/pull/210) to be notified when it becomes available.

**Reply by rhoboat** [reply]
[Commented](https://github.com/gruntwork-io/terraform-aws-vpc/pull/210#issuecomment-1163413327) there with an estimate!
